resources/assets/uad_lists.json

```
@@ -8003,7 +8003,7 @@
   {
     "id": "com.oneplus.opsports",
     "list": "Oem",
-    "description": "Cricket Scores (https://play.google.com/store/apps/details?id=com.oneplus.opsports)\nLets you access and follow cricket teams and tournaments.",
+    "description": "Cricket Scores (https://play.google.com/store/apps/details?id=com.oneplus.opsports)\nLet's you access and follow cricket teams and tournaments.",
```

```suggestion
    "description": "Cricket Scores (https://play.google.com/store/apps/details?id=com.oneplus.opsports)\nLets you access and follow cricket teams and tournaments.",
```
Incorrect. "Lets" is correct: https://www.grammarly.com/blog/lets-vs-lets/#:~:text=The%20difference%20between%20let's%20and%20lets&text=Let's%20is%20a%20contraction%20of%20%E2%80%9Clet%20us.%E2%80%9D%20You%20use,to%20allow%20or%20give%20permission.